### ISSUE TITLE
GL estimation-stuck at "this is not an estimate of the memory usage"

### ISSUE THREAD

**Issue by yzongzjnu** [ANGSD/angsd#212]
Hi all, 
I encountered a problem when using ANGSD to estimate genotype likelihood. I used the following command to do the estimation.

`SGE_Batch -c '/angsd/path/angsd -P 50 -b DoB_140_samlist -ref /reference/path/genomes.fasta -anc /anc/genomes.fasta -GL 1 -out DoB_140_AllChrom_strict -doGlf 2 -doMaf 2 -SNP_pval 1e-6 -uniqueOnly 1 -remove_bads 1 -only_proper_pairs 1 -trim 0 -C 50 -baq 1 -minMapQ 30 -minQ 20 -setMinDepth 4 -setMaxDepthInd 60 -doCounts 1 -doMajorMinor 4 -skipTriallelic 1 ' -r angsd.SGE -P 50 -m 200G`

It is still running even though 24 hours passed. I suspect it is abnormal due to the output files *.beagle.gz, *.mafs.gz, and *.arg were not updated for 10 hours. The following is execute log, wish it will be helpful to do trouble shooting.

```
Problem with difference in zlib version used for compiling and linking
-> ZLIB_VERSION: 1.2.7 zlibversion: 1.2.8
-> angsd version: 0.921-10-g2d8881c (htslib: 1.8-1-g5b60dd3) build(Apr  9 2018 06:56:37)
-> Reading fasta: /path/to/the/fasta
-> Reading fasta: /path/to/the/fasta
-> SNP-filter using a pvalue: 1.000000e-06 correspond to 23.928127 likelihood units
-> Parsing 140 number of samples
-> Allocated ~ 10 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 20 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 30 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 40 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 50 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 60 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 70 million nodes to the nodepool, this is not an estimate of the memory usage
-> Printing at chr: Fvb1-1 pos:2257300 chunknumber 100 contains 6719 sites
-> Printing at chr: Fvb1-1 pos:4545747 chunknumber 200 contains 4819 sites
-> Allocated ~ 80 million nodes to the nodepool, this is not an estimate of the memory usage
-> Printing at chr: Fvb1-1 pos:7116043 chunknumber 300 contains 14080 sites
-> Allocated ~ 90 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 100 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 110 million nodes to the nodepool, this is not an estimate of the memory usage
……..
-> Allocated ~ 310 million nodes to the nodepool, this is not an estimate of the memory usage
-> Allocated ~ 320 million nodes to the nodepool, this is not an estimate of the memory usage
```
Anybody who can give me some suggestions? Thanks in advance!

**Comment by yzongzjnu** [comment]
Stuck again even though there is no "Allocated ~ 320 million nodes to the nodepool, this is not an estimate of the memory usage" such error report after I rerun the program for 12 hours. It seems ANGSD will pause at the same running time. The output files again did not fresh for 10 hours.

**Comment by ANGSD** [comment]
Dear Yzong, are you only using this contig/chromosome:
'angsd.SGE'

Could you also try to limit the number of cores allocated to the program,
like -P 8. Maybe the program is stuck in some race condition.

Best

Den tir. 19. mar. 2019 kl. 17.28 skrev yzong <notifications@github.com>:

**Comment by yzongzjnu** [comment]
Thank you for your answer.  I apologize for posting the ambiguous command. Actually, I ran angsd using cluster. The  "-r angsd.SGE" is my running log folder rather than contig/chromosome I used for angsd analysis.  

I am shooting the trouble by running angsd on each chromosome separately. Maybe the problem is caused by one or several sites on a chromosome. Exactly, angsd analysis which was performed on subchromosome 6-1 will stop. I am using 1M windows on subchromosome 6-1 to  filter the exact region/sites which block the normally running of angsd. I will post here after I get the conclusion.
